--- pciutils-3.10.0/Makefile~	2023-05-01 12:59:54.000000000 +0000
+++ pciutils-3.10.0/Makefile	2023-05-30 15:26:02.030666362 +0000
@@ -19,7 +19,7 @@
 DNS=
 
 # Build libpci as a shared library (yes/no; or local for testing; requires GCC)
-SHARED=no
+SHARED=yes
 
 # Use libkmod to resolve kernel modules on Linux (yes/no, default: detect)
 LIBKMOD=
@@ -32,14 +32,14 @@
 ABI_VERSION=3
 
 # Installation directories
-PREFIX=/usr/local
+PREFIX=/usr
 BINDIR=$(PREFIX)/bin
 SBINDIR=$(PREFIX)/sbin
 SHAREDIR=$(PREFIX)/share
 IDSDIR=$(SHAREDIR)
 MANDIR:=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
 INCDIR=$(PREFIX)/include
-LIBDIR=$(PREFIX)/lib
+LIBDIR=$(PREFIX)/lib64
 PKGCFDIR=$(LIBDIR)/pkgconfig
 
 # Commands
@@ -49,7 +49,7 @@
 STRIP=--strip-program $(CROSS_COMPILE)-strip
 CC=$(CROSS_COMPILE)gcc
 else
-STRIP=-s
+STRIP=
 CC=cc
 endif
 AR=$(CROSS_COMPILE)ar
