diff --git a/Makefile b/Makefile
index be2359301f3e..adbcc64bb4f2 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ ZLIB=
 DNS=
 
 # Build libpci as a shared library (yes/no; or local for testing; requires GCC)
-SHARED=no
+SHARED=yes
 
 # Use libkmod to resolve kernel modules on Linux (yes/no, default: detect)
 LIBKMOD=
@@ -32,20 +32,20 @@ HWDB=
 ABI_VERSION=3
 
 # Installation directories
-PREFIX=/usr/local
+PREFIX=/usr
 BINDIR=$(PREFIX)/bin
 SBINDIR=$(PREFIX)/sbin
 SHAREDIR=$(PREFIX)/share
 IDSDIR=$(SHAREDIR)
 MANDIR:=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
 INCDIR=$(PREFIX)/include
-LIBDIR=$(PREFIX)/lib
+LIBDIR=$(PREFIX)/lib64
 PKGCFDIR=$(LIBDIR)/pkgconfig
 
 # Commands
 INSTALL=install
 DIRINSTALL=install -d
-STRIP=-s
+STRIP=
 ifdef CROSS_COMPILE
 STRIP+=--strip-program $(CROSS_COMPILE)strip
 CC=$(CROSS_COMPILE)gcc
